index.html

```diff
@@ -1226,6 +1244,52 @@
       <a href="#steps-receiving">Receiving and parsing content</a>
       and <a>Writing or pushing content</a> sections.
     </p>
+    <p>
+      To <dfn>convert an NDEFRecord object's bytes</dfn> to a given <var>type</var>, run these steps:
+    </p>
+    <ol>
+      <li>Let <var>bytes</var> be the <a>NDEFRecord</a> object's associated <a>bytes</a>.
```

Comment:
Do we really need to say object here? @marcoscaceres?

Reply:
Oh, you should change the algorithm to take a variable for the NDEFRecord. 

So:

```
To <dfn>convert an `NDEFRecord`</dfn>, pass a <var data-type="NDEFRecord">record</var>` and a <var>type</var>,  and run these steps:
```

Reply:
Add the `data-type` for `<var>type</var>` too, if you can.

Reply:
Thanks, done.